apps/nextjs/src/pages/index.tsx

```diff
@@ -35,100 +35,6 @@ export default function App() {
       >
         click to clear nexys
       </button>
-      {/*<button
```

Comment:
Nexys'deki /app klasöründeki repoları (örn. reactjs, nodejs) nexys'in testleri için kullanıyoruz.

Reply:
Bu sebep ile, butonların kaldırılmaması taraftarıyım. Örneğin nexys'in getLogPool özelliğini kullanmak istersem tekrar eklemem gerekecek.